monitoring/scripts/track-cid-backfill/python/main.py

```diff
@@ -0,0 +1,62 @@
+
+import os
+import pandas as pd
+
+csv_path = os.path.abspath(os.path.join(os.getcwd(), '..', 'local-csv'))
+# csv_path = os.path.abspath(os.path.join(os.getcwd(), '..', 'production-csv'))
```

Comment:
nit: lots of commented out code here etc

Reply:
ya that's just cause the script is super janky. That code is meant to be commented and uncommented depending on the env

Reply:
yeh i asked @jonaylor89 to prioritize shipping and not spend time cleaning up the backfill code since it just needs to run once as a standalone script